src/app.module.ts

```diff
@@ -42,7 +43,7 @@ import { Unsubscribe } from './notifications/v2/unsubscribe/unsubscribe.entity';
                         SavedSearchNotification,
                     ],
                     synchronize: false,
-                    ssl: process.env.DATABASE_SSL === 'true' ? true : false,
+                    ssl: process.env.DATABASE_SSL === 'true',
```

Comment:
Was this flagged up by analysis? I believe it's mapping an env string to a boolean

Reply:
Analysis picked this up as a way to simplify the statement. It checks the equality and assigns the same value the same way that the ternary would have I think.